### ISSUE TITLE
OpenBOR Wii doesn't work on USB device

### ISSUE THREAD

**Issue by SieKensou** [DCurrent/openbor#34]
It seems that it doesn't load the pak files from a USB device. I tried a with a pendrive and with a HDD and it crashes when I try to load OpenBOR ("Exception DSI Occurred!" screen or just a black screen sometimes).

**Comment by whitedragon0000** [comment]
no support yet... 
may Plombo can see...

**Comment by Plombo** [comment]
It used to work! I'll take a look and see what's up.

**Comment by SieKensou** [comment]
Okay, good, thanks. I hope you can find the problem.

**Comment by SieKensou** [comment]
Any update about this? Have you found the problem?

**Comment by Plombo** [comment]
I said I'd take a look, but then I got distracted and forgot about it...sorry about that.

**Comment by Plombo** [comment]
Okay, reproduced the issue. Don't know what's causing it yet, though.

**Comment by SieKensou** [comment]
Ah, okay, don't worry. Thanks for check this. I hope you can find what is causing this problem and a way to solve it. Please, keep me informed.

**Comment by Plombo** [comment]
Spent a few more minutes on this today but still haven't made much progress.

**Comment by whitedragon0000** [comment]
I wrote some changes to code..
try it compiling by yourself...

**Comment by SieKensou** [comment]
I have tested the last version (6276 from Jul,9 2018) and the problem is still there.... First try, the black screen with adressess and numbers error and the second try just a black screen.

**Comment by SieKensou** [comment]
Okay, thank you for the try.
The problem is that it doesn't load nothing, it doesn't show the menu with the list of pak files, it just crashes when I try to load the app. Plombo said he reproduced the issue so it seem it's some kind of bug. I hope this can be fixed to enjoy this great project. What is weird is that anybody said nothing about this bug until I decided to do it (maybe people has old versions that works fine and they didn't update to the last). For me, it didn't worked since the first one I have tried (although I don't remember wich version was).
